app/assets/stylesheets/application.scss

```
@@ -0,0 +1,140 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ */
+
+/* Custom bootstrap variables must be set or imported *before* bootstrap. */
+@import "bootstrap";
+
+@import "**/*";
+
+body {
+  margin: 0;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: #212529;
+  text-align: left;
+  background-color: #fff;
+}
+
+body, h1, h2, h3, h4, h5 {
+  font-family: "Gudea", sans-serif;
+}
+
+main {
+  padding: 2rem;
+}
+
+footer {
+  text-align: center;
+  color: #26A69A;
+}
+
+h1, h2, h3, h4 {
+  font-weight: bold;
+}
+
+a, h2, h3 {
+  color: #26A69A;
+}
+
+a:hover {
+  text-decoration: none;
+  color: #FF5722;
+}
+
+.btn:hover {
+  background-color: #FF5722;
+  border-color: #FF5722;
+}
+
+ul {
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  padding-inline-start: 40px;
```

It's generally preferred to just say 0 in cases like this like you do down below.  Also, since `margin-block-start` and friends are experimental I'd recommend just using the regular `margin-top` and such.  It's also much shorter to use the `margin` shorthand syntax:

```suggestion
  margin 1em 0; /* Equivalent to: margin 1em 0 1em 0; */
  padding-left: 40px;
```